backend/migrations/20230804194015_add_user_to_message.up.sql

```diff
@@ -0,0 +1,6 @@
+-- Add up migration script here
+ALTER TABLE message
+ADD COLUMN user_id BIGINT NOT NULL,
+ADD COLUMN channel_id BIGINT NOT NULL,
+ADD CONSTRAINT fk_user FOREIGN KEY (user_id) REFERENCES fluke_user(id),
+ADD CONSTRAINT fk_channel FOREIGN KEY (channel_id) REFERENCES channel(id);
```

Comment:
```suggestion
ADD CONSTRAINT FOREIGN KEY (user_id) REFERENCES fluke_user(id),
ADD CONSTRAINT FOREIGN KEY (channel_id) REFERENCES channel(id);
```
you dont need to have these names do you?